docs/sources/flow/getting-started/migrating-from-operator.md

```diff
@@ -51,13 +51,13 @@ This guide will provide some steps to get started with Grafana Agent for users c
     helm repo update
     ```
 
-4. Create a helm release. You may name the release anything you like. Here we are installing a release named `grafana-agent-metrics` in the `monitoring` namespace.
+4. Create a Helm release. You may name the release anything you like. Here we are installing a release named `grafana-agent-metrics` in the `monitoring` namespace.
 
-    ```
+    ```shell
     helm upgrade grafana-agent-metrics grafana/grafana-agent -i -n monitoring -f values.yaml --set-file agent.configMap.content=agent.river
     ```
 
-    This command uses the `--set-file` flag to pass the config file as a helm value, so that we can continue to edit it as a regular river file.
+    This command uses the `--set-file` flag to pass the configuration file as a Hhelm value, so that we can continue to edit it as a regular River file.
```

Comment:
```suggestion
    This command uses the `--set-file` flag to pass the configuration file as a Helm value, so that we can continue to edit it as a regular River file.
```

Ooops.. didn't delete the extra h